src/task/mod.rs

```diff
@@ -57,6 +58,7 @@ pub trait Task: Send + Sync + std::marker::Sized {
         max_retry_delay: None,
         retry_for_unexpected: None,
         acks_late: None,
+        content_type: Some(MessageContentType::Json),
```

Comment:
All defaults should be `None`

```suggestion
        content_type: None,
```